pkg/apis/crunchydata.com/v1/cluster.go

```diff
@@ -100,7 +100,7 @@ type PgclusterSpec struct {
 	UserSecretName      string                   `json:"usersecretname"`
 	RootSecretName      string                   `json:"rootsecretname"`
 	PrimarySecretName   string                   `json:"primarysecretname"`
-	CollectSecretName   string                   `json:"collectSecretName"`
+	ExporterSecretName  string                   `json:"exporterSecretName"`
```

Comment:
@tjmoore4 Is this captured in the upgrade?

Reply:
For the time being, this will be reverted to 'CollectSecretName' to support the current upgrade process. 
Hope to address this in the future.